eth_crowdfund_be/eth_crowdfund_api/src/models/Campaign.py

```diff
@@ -0,0 +1,86 @@
+from marshmallow import fields, Schema 
```

Comment:
"marshmallow is an ORM/ODM/framework-agnostic library for converting complex datatypes, such as objects, to and from native Python datatypes" - internet